client/src/Classifier.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef classifier_h
+#define classifier_h
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/ml/ml.hpp>
+
+using namespace std;
+using namespace cv;
+
+class Classifier {
+  CvSVM svm;
+
+public:
+  Classifier(string filename) {
+    svm.load(filename.c_str());
+  }
+
+  // Guess the characters name from the image icon.
+  string classify(const Mat &image) {
```

Comment:
The argument about *Is pointer / reference part of the type or a part of the binding* is probably as old as tabs v. spaces. But fine, for consistency, let's make it `Mat& image`.